### ISSUE TITLE
WRF+ROMS coupling

### ISSUE THREAD

**Issue by wjlyx** [jcwarner-usgs/COAWST#248]
Hello, I would like to ask, in the coupling experiment of roms+wrf, do we need to enable sst_update in the namelist.input of wrf? Thank you.

**Comment by jcwarner-usgs** [comment]
yes.   sst_update tells WRF to update the SST.  it does not matter where the sst is coming from, it tells WRF to update the values as the model runs.
